apps/users/tests/test_google_pub_sub.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from google.cloud.pubsub_v1 import futures
+
+from apps.users.google_pub_sub import GoogleCloudPubSubPublisher, Message
+
+
+class MessageTests(TestCase):
+    def test_encodes_string_to_bytes(self):
+        assert isinstance(Message("irrelevant").data, bytes)
```

Comment:
Notwithstanding the fact that there's a bunch of existing test code that uses unitttest, when possible we're trying to move toward using pytest-based tests instead.  Here, can easily do that, and also do away with the class, since there's only a single test function. 

```suggestion
def test_message_encodes_string_to_bytes(self):
        assert isinstance(Message("irrelevant").data, bytes)
```